### ISSUE TITLE
scATAC-seq level 1 data curator testing

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/schematic#292]
- [ ] Check if the generated template and validation in the data curator matches the scATAC-seq RFC requirements.

**Comment by milen-sage** [comment]
@xdoan the scATAC-seq template is ready to go in prod. 

The only people that can further verify the requested metadata works well for their actual data submission are data contributors (and some of them already requested trying that).

@xdoan, could you add scATAC-seq to the prod drop down? (I don't think we need to wait for all recently added data types to be verified, to start adding the templates one by one as we confirm they are ready).

**Comment by xdoan** [comment]
testing generation/validation/errors on dev and will add to prod once it passes

**Comment by xdoan** [comment]
I am seeing columns on the manifest that are not in the the RFC, Attribute Table, or in the Changelog. If they were added after the RFC was closed can you add them to the Changelog? In lieu of having the official data standard the RFC is the place to look up attribute descriptions, and some like, "Nuclei Barcode Length | Nuclei Barcode Read" don't have any descriptions when you hover over them in the spreadsheet and are not in the RFC with a description either. 

https://docs.google.com/spreadsheets/d/13wmIocQtg8bwWXUB_GygUJwQfHBfAliKi7Ln5c964WE/edit#gid=0

https://docs.google.com/spreadsheets/d/1soV03G4-rXiG-XAzbs6CrIvz-n2RJc6jRajyNsGY_Mo/edit#gid=0

**Comment by milen-sage** [comment]
These are added to the manifest because they are required columns specified for the attribute Nucleus Identifier in the RFC (also here: https://docs.google.com/spreadsheets/d/1soV03G4-rXiG-XAzbs6CrIvz-n2RJc6jRajyNsGY_Mo/edit#gid=0&range=12:12)

True, in follow up revisions the RFC editors need to add descriptions for these columns and define them as separate attribtues; for now I assumed most people working with epigenomics know what these columns mean (so the description is the most generic possible...). If we start getting questions I will be reaching out to the co-editors on this RFC.

**Comment by xdoan** [comment]
Yes I understand that you have added them for valid reasons, those reasons just are not captured in the RFC or in the Changelog. So when I go to validate it I have no idea what is the source of truth. When I look at the manifest there are very different fields in the manifest than in the RFC. For example if you change the names of an attribute, add things like, "scATAC-seq Read1,  scATAC-seq Read2" and leave out things like, "Median Fraction of Reads in Peaks, Median Passing Read Percentage, Peaks calling software, Median Percentage of Mitochondrial Reads per Nucleus, Threshold for Minimum Passing Reads" someone who signed off months ago would be very confused, and there is no record of the change. I'm sure you had discussed the requirement to have to describe Read1 and Read2 for the same file for example, but there is no explanation shared. Even if people are experts and understand the fields, if that was not communicated to them in a form they could refer to (the RFC since we don't have the data standard yet) then I think they would get confused. 

Wrt to "Nuclei Barcode Length | Nuclei Barcode Read" which are freetext fields with no descriptions, I feel like people could describe the same attribute different ways because there is no example. Even if they understand it, I looked for a description to see if I should use "5/five" etc. 

Also Column L & M have the same description, and K seems to be describing something else. 

If we are releasing this to prod to beta test with people we have established trust with, who we know will give us feedback, and who have open communication channels that is different than if we want a well documented production QCd template that stands on its own to a more naive user.

**Comment by milen-sage** [comment]
Hmm it looks like the attribute table is different than the RFC table. I'll add the missing fields and will correct the description - good catch. 

I can add description to Nuclei Barcode Length, Nuclei Barcode Read; though that's not been provided in the RFC either, so hopefully the description would be accurate.

**Comment by milen-sage** [comment]
see updated PR #294

**Comment by xdoan** [comment]
thanks for adding those missing columns. 

are we also accepting fastqs for this? If so the system you have set up to make both "scATAC-seq Read1 | scATAC-seq Read2" required with no NA option does not make sense.

**Comment by milen-sage** [comment]
let me double check - since we have similar options for scRNA-seq level 1 and we are accepting fastq's there.

**Comment by milen-sage** [comment]
we are good with the way the two reads are currently defined for scATAC-seq

**Comment by xdoan** [comment]
ok

**Comment by milen-sage** [comment]
for reference 

https://divingintogeneticsandgenomics.rbind.io/post/understand-10x-scrnaseq-and-scatac-fastqs/

**Comment by xdoan** [comment]
approved #294
